### DISCUSSION TITLE
Test

### DISCUSSION THREAD

**Discussion by Exo-Boom** [vsb-vaj/.github#2] [Q&A] [answered]
Dobrý den,

mohl bych se zeptat na kdy je plánovaný test?
A bude spíše praktický nebo teoretický?

**Comment by petrvecera** [comment, marked as answer]
Test je planovany na 9 tyden, nebudeme jej delat na stadartni hodinne ( mame jen 8 vyuc hodin ) aby kazdy mel dostatek casu. Tj nekdo ho muze udelat za 25 minut, nekdo muze klidne sedet hodinu a pul. 

Bude 25 otazek. 
- Odpovedi budou a/b/c/d  (easy mode) 
- Jedna spravna odpoved 

Mix teoretickych / praktickych znalosti. Spise ty prakticke. 
Napriklad: 
 - Jak vypada express route, pomoci ktere ziskame id  z `/shop/product/1231?theme=dark`
 - Jakou pouzijeme HTTP metodu abychom udelali pomoci REST api xxx
 - Co bude vysledkem nasledujicho kodu
 - Co je spatne na nasledujicim kodu 

Na testu budou pouze otazky na ktere jsou odpovedi ve znalostech z prezentaci / z ukolu ktere jsme delali na cvice nebo znalosti nutne ke splenani zadani projektu. 
Pokud nekdo delal vsechny ukoly na cvikach / precte si prednasky, nemel by mit problem ziskat 25/25 bodu.  Min je 10b. 

Bude 1 opravny pokus. Nicmene nepredpokladam, ze by nekdo neziskal alespon 10 😄
